.circleci/config.yml

```diff
@@ -137,7 +137,7 @@ jobs:
                --label "build.url=$CIRCLE_BUILD_URL" \
                --label "build.gitref=$CIRCLE_SHA1"
 
-            if [[ "${CIRCLE_BRANCH}" == "master" ]]; then
+            if [[ "${CIRCLE_BRANCH}" == "PIC-375-assume-IAM-role-rebased" ]]; then
```

Comment:
I guess you take this out now and revert back to master?

Reply:
yep will do a last commit before  merging once all the review changes  have been applied